lib/settings/update_screen.dart

```diff
@@ -0,0 +1,672 @@
+                    ),
+                    const SizedBox(height: 10),
+                    Row(
+                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                      children: [
+                        Expanded(
+                          child: ElevatedButton.icon(
+                            style: ElevatedButton.styleFrom(
+                              elevation: 3,
+                              minimumSize:
+                                  const Size.fromHeight(65), // Set height to 65
+                            ),
+                            onPressed: _viewChangelog,
+                            icon: const Icon(Icons.article),
+                            label: const Text('View Changelog',
+                                style: TextStyle(fontSize: 24)),
+                          ),
+                        ),
+                        const SizedBox(
+                            width: 10), // Add some space between the buttons
+                        Expanded(
+                          child: ElevatedButton.icon(
+                            style: ElevatedButton.styleFrom(
+                              elevation: 3,
+                              minimumSize:
+                                  const Size.fromHeight(65), // Set height to 65
+                            ),
+                            onPressed: () async {
+                              _performUpdate(context);
+                            },
+                            icon: const Icon(Icons.download),
+                            label: const Text('Download Update',
+                                style: TextStyle(fontSize: 24)),
+                          ),
+                        ),
+                      ],
+                    ),
+                  ] else ...[
+                    Row(
+                      children: [
+                        const Icon(Icons.check_circle,
+                            color: Colors.green, size: 30),
+                        const SizedBox(width: 10),
+                        Text(
+                            _betaUpdatesOverride
+                                ? 'Bleeding Edge is up to date!'
+                                : 'Orion is up to date!',
+                            style: const TextStyle(
+                                fontSize: 26, fontWeight: FontWeight.bold)),
+                      ],
+                    ),
+                    const Divider(),
+                    Text(
+                        _betaUpdatesOverride
+                            ? 'Current Version: $_currentVersion ($_release)'
+                            : 'Current Version: ${_currentVersion.split('+')[0]}',
+                        style: const TextStyle(fontSize: 20)),
+                  ],
+                ],
+              ),
+            ),
+          ),
+          // Placeholder for Odyssey updater - pending API changes
```

Comment:
I'd also drop a TODO, just so it's easier to find/highlight that things that remain un-done

Reply:
Resolved by [811e9fc](https://github.com/TheContrappostoShop/Orion/pull/14/commits/811e9fca935c91187b64a20e65b130c7424ae331)